phpBB/develop/adjust_avatars.php

```diff
@@ -30,6 +30,11 @@
 	die('database not up to date');
 }
 
+if (!isset($config['storage\\avatar\\config\\path']) || $config['storage\\avatar\\config\\path'] != 'phpbb\\storage\\provider\\local')
+{
+	die('use local provider');
```

Comment:
This is not really a meaningful error message.

Reply:
Uhm, this is from the other pr, it will disapear when the other pr is acepted and i do the rebase